src/aap_eda/core/tasking/__init__.py

```diff
@@ -41,6 +47,76 @@
 ]
 
 
+def enable_redis_prefix():
+    if rq.worker_registration.REDIS_WORKER_KEYS != "rq:workers":
+        # changes have been made
+        return
+
+    try:
+        redis_prefix = f"EDA{service_id()[0:8]}-rq"
```

Comment:
separator between EDA and service_id
```suggestion
        redis_prefix = f"EDA-{service_id()[0:8]}-rq"
```